src/modules/program/dtos/create-program/create-program.schema.ts

```diff
@@ -0,0 +1,13 @@
+import Joi from "joi";
+import {CreateProgramDto} from "./create-program.dto";
+
+
+export const CreateProgramValidationSchema = Joi.object<CreateProgramDto>({
+  name: Joi
+    .string()
+    .required()
+    .min(1)
+    .message('Name must be between 1 and 50 characters long')
```

Comment:
Move text to error messages?

Reply:
This makes no sense, the Joi validator assumes that there will be text ops in the scheme, and all business errors in a separate object